runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/DefaultPacBio.java

```diff
@@ -213,6 +214,13 @@ protected StatusResponse getStatus(String url) {
     return new RestTemplate().getForObject(url, StatusResponse.class);
   }
 
+  protected String getSampleSheet(String url) {
+    return new RestTemplate().getForObject(url, String.class);
+  }
+
+  private final static Pattern LINES = Pattern.compile("\\r?\\n");
+  private final static Pattern WELL_LINE = Pattern.compile("^([A-Z]\\d+),.*$");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)